vsphere/tests/legacy/test_vsphere.py

```diff
@@ -551,7 +551,7 @@ def test__collect_metrics_async_hostname(vsphere, instance, aggregator):
     aggregator.assert_metric('vsphere.mymetric', value=23.4, hostname="foo")
 
 
-def test_check(vsphere, instance):
+def test_check_run(vsphere, instance):
```

Comment:
Note for reviewers: renamed to make it easier to target only this test. Currently, `pytest -k test_check` will run `test_check.py` file.